### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by othmanhicheur**

### CODE REVIEW

File: paradigm_client/langchain_client.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+from langchain.llms.utils import enforce_stop_tokens
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from paradigm_client.communicator import SagemakerCommunicator
+from paradigm_client.remote_model import RemoteModel
+from paradigm_client.request import CreateParameters
+
+
+class MiniLLM(LLM):
+    def __init__(self, host, paradigm_api_key, verbose, **kwargs):
+        os.environ['PARADIGM_API_KEY'] = paradigm_api_key
```
Comment (iacolippo) [inline paradigm_client/langchain_client.py:17]: can you remove `paradigm_api_key` from the arguments of `__init__` and do something like

```
if os.environ.get("PARADIGM_API_KEY", None) is None:
    warnings.warn("No PARADIGM_API_KEY environment variable detected")
```

File: paradigm_client/langchain_client.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            "biases": self.biases,
+            "stop_regex": self.stop_regex,
+            "prettify": self.prettify,
+            "return_log_probs": self.return_log_probs,
+            "echo": self.echo,
+        }
+
+    @property
+    def _llm_type(self) -> str:
+        return "quadriton"
```
Comment (iacolippo) [inline paradigm_client/langchain_client.py:67]: call this "paradigm" instead

File: paradigm_client/langchain_client.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    def _call(
```
Comment (iacolippo) [inline paradigm_client/langchain_client.py:69]: do you have a small test that calls this object?

Comment (othmanhicheur) [reply paradigm_client/langchain_client.py:69]: what do you mean ?

Comment (iacolippo) [reply paradigm_client/langchain_client.py:69]: a function that sends a text and checks that it correctly receives a text back from this object.

Also why did you resolve the other comments? :eyes:

Comment (othmanhicheur) [reply paradigm_client/langchain_client.py:69]: I think that I resolve other comments in order to not be lost. 

For the test, I will do that !

**Pull request closed without merging**